homeassistant/components/dwd_weather_warnings/sensor.py

```diff
@@ -80,7 +81,7 @@ class DwdWeatherWarningsSensor(
     def __init__(
         self,
         coordinator: DwdWeatherWarningsCoordinator,
-        entry: ConfigEntry,
+        entry: DwdWeatherWarningsConfigEntry,
```

Comment:
FYI this is also not strictly necessary. You can always pass a more concrete type. E.g. when `ConfigEntry` is expected, it's perfectly valid to pass `DwdWeatherWarningsConfigEntry` to it instead.

Although it doesn't hurt either in this case 😃

Reply:
but with this i was able to remove the import of `ConfigEntry` here 😁